build.gradle.kts

```diff
@@ -15,7 +15,7 @@ plugins {
 
 
 group = "io.outfoxx"
-version = "1.5.0-SNAPSHOT"
+version = "1.5.1-SNAPSHOT"
```

Comment:
Please don't commit changes to the version. Versions are handled during releases.